wsgi/odyssy/odyssy/settings.py

```diff
@@ -51,9 +51,11 @@
 INSTALLED_APPS = (
     'photologue',
     'sortedm2m',
+    'taggit',
     'events.apps.EventConfig',
     'news.apps.NewsConfig',
     'basic.apps.BasicConfig',
+    'tag.apps.TagConfig',
```

Comment:
This is for the tag link app. taggit itself handles all the functionalities of tagging things, but to represent through views i had added this thing. for the `/tag/<tag>/` link.